benchmarks/basic/gocql/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang
+
+# Copy benchmark code into the container
+COPY source /source
+WORKDIR /source
+
+# Compile the code
+RUN go build -o basic .
```

Comment:
I don't know how go works but Rust required a `--release` flag for best performance. And other benchmarks have link time optimalization (lto) enabled.

Reply:
go compiler optimizes by default

Reply:
oh ok cool